`make install` fails on build/po/cs.gmo

**Affects versions :**
 - OS: Linux Ubuntu 18.04 X11
 - Libgtk: 
```bashii  libgtk2.0-dev                                 2.24.32-1ubuntu1                                   amd64        development files for the GTK+ library
ii  libgtk-3-dev:amd64                            3.22.30-1ubuntu4                                   amd64        development files for the GTK+ library
ii  libgtkmm-3.0-dev:amd64                        3.22.2-2                                           amd64        C++ wrappers for GTK+ (development files)
```
 - Version of Xournal++ master c533ab9bc1de77ca52d542491f03e8a4760e0f7d

**Describe the bug**

After a successfully `make`, 

    make install

fails with: 

```bash
CMake Error at po/cmake_install.cmake:41 (file):
  file INSTALL cannot find "/home/romano/software/xournalpp/build/po/cs.gmo".
Call Stack (most recent call first):
  cmake_install.cmake:102 (include)


Makefile:85: recipe for target 'install' failed
make: *** [install] Error 1
```

**To Reproduce**

    rm -r build
    git reset --hard 
    git checkout master 
    git pull
    mkdir build 
    cd build 
    cmake ../
    make
    make install

**Expected behavior**

`xournalpp` installs in `/usr/local/...` 

Thanks!!!

--- a/po/CMakeLists.txt
+++ b/po/CMakeLists.txt
@@ -20,7 +20,7 @@ if (XGETTEXT_FOUND)
       "${PROJECT_SOURCE_DIR}/ui/*.glade"
   )
 
-  gettext_create_translations ("${potfile}")
+  gettext_create_translations ("${potfile}" ALL NOUPDATE)
 
 else (XGETTEXT_FOUND)
   message (ERROR "Gettext not found!")

